src/integrationTest/resources/legal-advisor-service-decision-deemed-not-granted-response.json

```diff
@@ -10,12 +10,27 @@
           "receivedServiceAddedDate": "${json-unit.any-string}",
           "serviceApplicationGranted": "No",
           "serviceApplicationDecisionDate": "${json-unit.any-string}",
-          "deemedServiceDate": "${json-unit.any-string}",
-          "serviceApplicationOutcomeLabel":"${json-unit.any-string}"
+          "serviceApplicationOutcomeLabel":"${json-unit.any-string}",
+          "serviceApplicationRefusalReason": "${json-unit.any-string}"
```

Comment:
are these dynamic ?

Reply:
Dont think serviceApplicationOutcomeLabel is but serviceApplicationRefusalReason is dynamic. 
serviceApplicationRefusalReason will have value only if the service is refused

Reply:
test data correction done